docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Explainer: what is a jumpbox or bastion host?"
+description: Defines the terms 'jumpbox' and 'bastion host'
+author: telmosampaio
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
+Different users need to access these separate VMs in a secure fashion, without you necessarly providing direct access to the VMs themselves. You can do that by using a **jumpbox**, **bastion host**, or **secure administrative host**. 
+
+> [!NOTE]
+> The terms **jumpbox**, **bastion host**, and **secure administrative host** are used interchangebly in the industry. We will use the term **jumpbox** in this document to facilitate reading.
+
+A **jumpbox** is a computer used to manage devices in different security zones. In public clouds, jumpboxes become extremelly valuable as a way to manage VMs that are protected from access from the public Internet. The jumpbox VM itself might be exposed to the Internet, and have access to VMs in your workload that are not exposed to the public. That creates an isolation boundary, or **DMZ**, that you control to provide management access to secure resources.
```

Comment:
"In public clouds, a jumpbox is valuable as a way to manage VMs..."

You may also consider something like:

"The jumpbox sits between the unprotected network and the private secure network. Access to the VMs in the private secure network is limited to that which the jumpbox provides. This narrows the potential attack surface to only that which the jumpbox allows, creating an isolation boundary...."